pkg/route53/route53.go

```diff
@@ -57,3 +59,51 @@ func (r *Route53) PublicHostedZones() ([]*HostedZone, error) {
 
 	return zones, nil
 }
+
+// DeleteHostedZoneRecordSets removes all recods in a hosted zone except NS and SOA records, to enable zone deletion
+func (r *Route53) DeleteHostedZoneRecordSets(hostedZoneID string) (*route.ChangeResourceRecordSetsOutput, error) {
```

Comment:
This makes me really nervous, I honestly believe we should introduce `okctl` tags in the records that we manage and only delete those. We do not want to end up in a situation where we wipe out all the records in a users hostedzone by mistake